pkgs/development/php-packages/composer/default.nix

```
@@ -1,13 +1,11 @@
 { mkDerivation, fetchurl, makeWrapper, unzip, lib, php }:
-let
+
+mkDerivation rec {
   pname = "composer";
   version = "2.2.7";
-in
-mkDerivation {
-  inherit pname version;
 
   src = fetchurl {
-    url = "https://getcomposer.org/download/${version}/composer.phar";
+    url = "https://github.com/${pname}/${pname}/releases/download/${version}/composer.phar";
```

```suggestion
    url = "https://github.com/composer/composer/releases/download/${version}/composer.phar";
```
to easily get to the repo

---

Why is this even done to begin with? The original URL should be the official upstream source, right?

---

Yeah I think it was something to do, but @SuperSandro2000 is right, there is no need to use those placeholders in the URL.